packages/fewer/src/Repository/index.ts

```diff
@@ -244,6 +259,26 @@ export class Repository<
     );
   }
 
+  whereFn(
```

Comment:
Can we just do this with an overload on where?

Reply:
No, breaks the extra property checking which is really useful.